MIGRATION.md

```diff
@@ -53,13 +60,59 @@ Tip: Provide the `--transform`/`-t` argument at the end of the command, so that
 
 ## From v2.x to v3
 
-(intro)
+Circuit v3 is a large major release, including long-awaited changes from the full year that passed since v2. This guide will help you upgrade your application. Don't hesitate to contact the [maintainers](https://github.com/sumup-oss/circuit-ui#maintainers) if you have any further questions.
+
+### Accessibility
+
+Any accessible label props that were previously optional are now required and enforced in all components.
+
+To help identify places where accessible labels are missing (even if an app is built with TypeScript, some configurations will not break the build if required props are missing in JavaScript files), components will throw runtime errors in development at missing labels. Production and testing builds are not affected.
+
+During the migration and while the missing labels are still being added, you can use an escape hatch to continue running the app in development without throwing accessibility errors.
+
+In your app, expose the `UNSAFE_DISABLE_ACCESSIBILITY_ERRORS` environment variable. You can use the [Webpack `DefinePlugin`](https://webpack.js.org/plugins/define-plugin/) ([here's an example](https://github.com/sumup-oss/circuit-ui/blob/main/.storybook/main.js#L45-L53) in the Circuit UI Storybook config) or, if your app uses Next.js, you can declare the variable in your `next.config.js` ([here's an example](https://github.com/sumup/ze-dashboard/blob/master/next.config.js#L78) in the SumUp merchant dashboard).
+
+Now, if you want to turn off the accessibility errors temporarily, run the development app with the environment variable set to `true`:
+
+```sh
+UNSAFE_DISABLE_ACCESSIBILITY_ERRORS=true yarn dev # or yarn start
+```
+
+Keep in mind that this escape hatch is not meant as a way to permanently avoid the errors, but as a temporary workaround while the missing labels are being written, localized and added to the relevant components.
+
+> Note: for the `Input` and `Select` components, use the built-in `label` prop instead of using the `Label` component separately.
```

Comment:
This seems out of place. This change was already done in v2. How about this:

```suggestion
> Reminder: For the `Input` and `Select` components, use the built-in `label` prop instead of using the `Label` component separately.
```